core_lib/interface/scribblearea.cpp

```diff
@@ -617,6 +621,9 @@ void ScribbleArea::mouseMoveEvent( QMouseEvent *event )
 
     currentTool()->mouseMoveEvent( event );
 
+    // used to clear and update canvas cursor
+    update();
```

Comment:
I've tried that but it's not fast enough to perceive the cursor if you move too fast and will therefore leave a fragments of the cursor. It doesn't matter how big the rect is.